doc/releases/release-notes-2.7.rst

```diff
@@ -2,24 +2,132 @@
 
 .. _zephyr_2.7:
 
-.. _zephyr_2.7.1:
+.. _zephyr_2.7.2:
 
-Zephyr 2.7.1
+Zephyr 2.7.2
 ####################
 
 This is an LTS maintenance release with fixes.
 
+Issues Fixed
+************
+
+These GitHub issues were addressed since the previous 2.7.1 tagged
+release:
+
+.. comment  List derived from GitHub Issue query: ...
+   * :github:`issuenumber` - issue title
+
+* :github:`44902` - x86: FPU registers are not initialised for userspace (eager FPU sharing)
+* :github:`44497` - Add guide for disabling MSD on JLink OB devices and link to from smp_svr page
+* :github:`38576` - net shell: self-connecting to TCP might lead to a crash
+* :github:`44886` - Unable to boot Zephyr on FVP_BaseR_AEMv8R
+* :github:`44531` - bl654_usb without mcuboot maximum image size is not limited
+* :github:`43858` - mcumgr seems to lock up when it receives command for group that does not exist
+* :github:`45869` - doc: update requirements
+* :github:`44352` - stm32l5x boards missing the openocd runner
+* :github:`45870` - drivers: virt_ivshmem: Allow multiple instances of ivShMem devices
+* :github:`44336` - nucleo_wb55rg: stm32cubeprogrammer runner is missing for twister tests
+* :github:`44337` - twister: Miss sn option to stm32cubeprogrgammer runner
+* :github:`44310` - net: gptp: type mismatch calculation error in gptp_mi
+* :github:`39882` - Bluetooth Host qualification on 2.7 branch
+* :github:`42622` - pm: pm_device structure bigger than nessecary when PM_DEVICE_RUNTIME not set
+* :github:`42825` - MQTT client disconnection (EAGAIN) on publish with big payload
+* :github:`43470` - wifi: esp_at: race condition on mutex's leading to deadlock
+* :github:`39184` - HawkBit hash mismatch
+* :github:`43490` - net: sockets: userspace accept() crashes with NULL addr/addrlen pointer
+* :github:`43853` - 2.7 branch: X86 MSI messages always get to BSP core (need a fix to be backported)
+* :github:`43807` - Test "cpp.libcxx.newlib.exception" failed on platforms which use zephyr.bin to run tests.
+* :github:`43580` - hl7800: tcp stack freezes on slow response from modem
+* :github:`42299` - spi: nRF HAL driver asserts when PM is used
+* :github:`43572` - [Backport v2.7-branch] stm32: wrong clock the LSI freq for stm32l0x mcus
```

Comment:
Do you want to scrub out these `[Backport]` sections?  I think this is the only one here.

Reply:
I thought I got them already - good catch ;-)